docs/gambit/mutation-verifier.md

```diff
@@ -35,129 +34,205 @@ If you are on Linux, you will need pip version 20.3 or above.
 
 ## Running the Mutation Verifier
 
-Once you have updated your `certora-cli` installation using `pip` to get the relevant
-dependencies, run Gambit from the command line:
+The mutation verification script is called `certoraMutate`.
+Run it from the command line:
 
-```
-certoraMutate --prover_conf path/to/prover/prover.conf --mutation_conf path/to/mutation/mutation.conf
+```sh
+certoraMutate --prover_conf path/to/prover.conf --mutation_conf path/to/mutation.mconf
 ```
 
+If you do, the script will generate code mutants, 
+  then submit a verification job per mutant to Certora's server.
+
 ```{note}
 You must run `certoraMutate` from the root of the Solidity project directory.
-The files `prover.conf` and `mutation.conf`
-can be in their own directories, but must always be within the project directory.
+The files `prover.conf` and `mutation.conf` can be in their own directories, 
+  but must always be within the project directory.
```

Comment:
Is it true now that conf files must be a sub-directory?
Is it true after we make the change to relative paths?

Reply:
Yes after that change the documentation must be updated